wolox-books/src/app/screens/auth/auth.guard.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
+import { Observable } from 'rxjs';
+import { UserService } from '../../components/user/user.service';
+import { Router } from '@angular/router';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthGuard implements CanActivate {
+
+  constructor(private userService: UserService, private router: Router) {  }
+
+  public canActivate() {
+    if(this.userService.loggedIn())
+      return true;
```

Comment:
If this is the only line executed after the `if`, write it in the same line of the `if`